testing/collatzConjecture.js

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai'
+import collatzConjecture from '../functions/collatzConjecture'
+
+
+describe('#collatzConjecture', function() {
+  context('takes an integer', function() {
+    it('should return an array of the Collatz Sequence of the input integer', function() {
```

Comment:
It's also a good idea to test your base case when you're writing recursive functions (e.g., `collatzConjecture(1)`).